libs/utils/utils.js

```diff
@@ -480,6 +481,13 @@ export function decorateAutoBlock(a) {
           parentElement.parentElement.replaceChild(div, parentElement);
         }
       }
+
+      // fragment url with mp4 - previewing a fragment page with mp4 video
```

Comment:
The first part of this comment is redundant 
```suggestion
      // previewing a fragment page with mp4 video
```